README.md

```diff
@@ -74,9 +74,16 @@ If the levers of capitalism place it in opposition to just data practices, can w
 
 **Readings:**
 
-Coming soon! Readings should appear at least 1 month before the date of discussion.
-
-Please feel free to suggest & recommend readings on the [Issue](https://github.com/datatogether/reading_datatogether/issues/41) for this session!
+1. Kathi Weeks (2011): **The Problem with Work p5-8, 42-47, 51-57** Available at: [libcom.org](https://libcom.org/files/the-problem-with-work_-feminism-marxism-kathi-weeks.pdf)
+1. Hanna Hurr (2016): **Silvia Federici interviewed in Mask Magazine** Available at: [maskmagazine.com](http://www.maskmagazine.com/the-control-issue/struggle/interview-silvia-federici)
```

Comment:
I think we usually show the full URL (e.g., "maskmagazine.com/the-control-issue/struggle/interview-silvia-federici"), I actually don't know why, I think a vestigial imagining as a potentially printed documents?